larq_compute_engine/mlir/tests/translate-tflite.mlir

```diff
@@ -0,0 +1,37 @@
+// RUN: lce-tf-opt %s -tfl-legalize-lce -lce-translate-tfl -verify-diagnostics | FileCheck %s
```

Comment:
Since we now test the full round-trip in this test already, how do you feel about consolidating this with the [`legalize-lce.mlir`](https://github.com/larq/compute-engine/blob/main/larq_compute_engine/mlir/tests/legalize-lce.mlir) tests?

This can be done via the `--check-prefix` option in the `FileCheck` which allows to run different passes on the same IR. Similar to how it is done [here](https://github.com/tensorflow/tensorflow/blob/f42ecd1bdcee0aea8a2afabbe699679dc07b8b9f/tensorflow/compiler/mlir/lite/tests/legalize-tf-while.mlir#L1-L3).

Reply:
That's a good idea! Let me make the change.